### ISSUE TITLE
Images in e-books do not work with optical character recognition using NVDA

### ISSUE THREAD

**Issue by cary-rowen** [edrlab/thorium-reader#2042]
I added the tabindex="0" attribute to the images in the Epub file and I want to be able to recognize the image objects using NVDA +R at any time when reading in thorium-reader.

When I do this, NVDA reports "Content is not visible".

Specifically, the following errors can be found in NVDA's logging:
```ValueError: Image not visible (invalid screen coordinates)```

**Comment by cary-rowen** [comment]
@danielweck 
I was wondering if you could have any thoughts on this?

**Comment by danielweck** [comment]
Thinking about this now, I am wondering if this is because the pagination view is activated instead of the scrolling mode that is typically equivalent to a traditional webpage rendered in a web browser. Either way, there is a possibility that the image is not "in view" inside the scrolling or paginated viewport, causing NVDA's feature to fail. At any rate, this is not something we can fix in Thorium so I am closing this issue. We would love to hear more about this interesting use-case though, so do not hesitate to write back here (we will see the notification)
